lib/internal/crypto/rsa.js

```diff
@@ -326,14 +326,7 @@ async function rsaImportKey(
         'NotSupportedError');
   }
 
-  if (algorithm.name === 'RSA-PSS') {
-    if (
-      keyObject.asymmetricKeyType !== 'rsa' &&
-      keyObject.asymmetricKeyType !== 'rsa-pss'
-    ) {
-      throw lazyDOMException('Invalid key type', 'DataError');
-    }
-  } else if (keyObject.asymmetricKeyType !== 'rsa') {
+  if (keyObject.asymmetricKeyType !== 'rsa') {
```

Comment:
Is there a test that covers this error case? If not, please add one.

Reply:
There is.